tests/laser/state/mstack_test.py

```diff
@@ -0,0 +1,68 @@
+from mythril.laser.ethereum.state import MachineStack
+from mythril.laser.ethereum.evm_exceptions import *
+from tests import BaseTestCase
+
+
+class MachineStackTest(BaseTestCase):
```

Comment:
Can you structure the unit tests in the following way (when possible):
# Arrange
# Act
# Assert

Not saying that this is perfect, but it will allow us to stick to an easy format that is simple to read and semi-selfdocumenting. Plus in this case you see that a lot of the testcases actually have multiple testcases, I think it makes more sense to have a separate unittest for each test case

Reply:
Good job :+1: